resharper/resharper-unity/src/CSharp/Daemon/UsageChecking/UsageInspectionsSuppressor.cs

```diff
@@ -92,13 +121,83 @@ public bool SuppressUsageInspectionsOnElement(IDeclaredElement element, out Impl
                     // But it still should be checked if the field is ever accessed from the code.
                     flags = ImplicitUseKindFlags.Assign;
                     return true;
-                
+
+                case IProperty property when IsImplicitlyUsedInterfaceProperty(property):
+                    flags = ImplicitUseKindFlags.Assign;
+                    return true;
+
```

Comment:
Merge this `case` with the one below: `case IProperty property when Is..() || Is()…:`